controllers/hivelocitymachine_controller.go

```diff
@@ -89,7 +89,7 @@ func (r *HivelocityMachineReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	// Fetch the Cluster.
 	cluster, err := util.GetClusterFromMetadata(ctx, r.Client, machine.ObjectMeta)
 	if err != nil {
-		log.Info("Machine is missing cluster label or cluster does not exist: %s", err) // question: added %s+err
+		log.Info("Machine is missing cluster label or cluster does not exist: %s", err)
```

Comment:
Using this kind of logging statement contradicts structured logging. Maybe you should use that instead.

Reply:
fixed, use key-value pairs now.